src/index.html

```diff
@@ -5,8 +5,320 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo"
+      >
+        <img 
+          src="images\logo.png" 
+          alt="logo"
+        >
+      </a>
+ 
+      <nav class="nav">
+        <a class="nav__item is-active" href="#">Apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <li>
          <a class="nav__item is-active" href="#">
            Apple
          </a>
        </li>
```
make them as list elements